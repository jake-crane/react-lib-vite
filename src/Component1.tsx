const Component1 = () => <div>Hello World!</div>;

export default Component1;