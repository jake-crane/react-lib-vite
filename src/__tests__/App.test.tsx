import { render } from "@testing-library/react";
import Component1 from "../Component1";


describe("<Component1 />", () => {
    test("should display a blank login form, with remember me checked by default", async () => {
        const { container } = render(<Component1 />)
        expect(container).toMatchSnapshot();
    });
});