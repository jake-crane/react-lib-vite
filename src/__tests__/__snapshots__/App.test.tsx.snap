// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component1 /> should display a blank login form, with remember me checked by default 1`] = `
<div>
  <div>
    Hello World!
  </div>
</div>
`;
